src/machines/modelingMachine.ts

```diff
@@ -3973,7 +3977,10 @@ export function isEditingExistingSketch({
   const variableDeclaration = getNodeFromPath<VariableDeclarator>(
     kclManager.ast,
     sketchDetails.sketchEntryNodePath,
-    'VariableDeclarator'
+    'VariableDeclarator',
+    false,
+    true // suppress noise because we know sketchEntryNodePath might not match up to the ast if the user changed the code
```

Comment:
This function is becoming a beast, 
I found out recently that it can be used to replace nodes in the tree, which is pretty far removed from the functions name
I think it needs to be changed to named params instead of ordered params,

Anyway this is the case where I expect a pathToNode to maybe fail, because if the user has deleted all their profiles, than sketchEntryNodePath which is a path to one of the profiles is going to fail, so I'm supressing the noise since this is expected.